app/assets/stylesheets/pages/homepage.css

```diff
@@ -0,0 +1,50 @@
+.banner-hero {
+  background-image: url("https://dm2305files.storage.live.com/y4mLTlsAOlVIbxayA4OuTrKgh5bhSouUHtvsZcoX-mh1AaD2CDTNG8b6-p5_1gwZML9FEpMOMjRKWGBskNVzDP1NtaFq4a7_R6D39cT8U7pMYOz3ys19QgQZwH4g6C0OGWv-CHTkXKYN58IpkKDjveBeCRT083wGENMnR7KsdfQ1T7xZdp0YXcj2fvBJiI9ZXVo?width=2482&height=3475&cropmode=none");
+  background-color: var(--light-beige);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  color: white;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.banner-hero__txt {
+  width: 60vw;
+}
+
+.banner-buttons {
+  background-color: var(--red);
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.banner-about {
+  background-color: var(--blue);
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+  gap: 30px;
```

Comment:
**suggestion:** this looks consistent with paddings. We should consider extracting spacing values to variables.